lib/assetcompute.js

```diff
@@ -102,6 +102,28 @@ class AssetCompute {
         }
     }
 
+    /**
+     * Unregister I/O events and journal
+     */
+    async unregister() {
+        const response = await fetch(`${this.url}/unregister`, {
+            method: "POST",
+            headers: {
+                authorization: `Bearer ${this.accessToken}`,
+                "x-gw-ims-org-id": this.org,
+                "x-ims-org-id": this.org,
+                "x-api-key": this.apiKey
+            },
+            retryOptions: this.retryOptions
```

Comment:
Make sure it does not retry if the error is a 404, which means no registration existed. Retry would be futile then. Or probably any 4xx error. See https://git.corp.adobe.com/nui/nui/blob/master/doc/api.md#unregister

Reply:
By default, node-fetch-retry only retries on 5xx errors: https://github.com/adobe/node-fetch-retry#optional-custom-parameters

We use the default retry options unless otherwise specified. but we can change this to not give the user a choice and just use the default values by completely removing the retryOptions from the request:
```
const response = await fetch(`${this.url}/unregister`, {
            method: "POST",
            headers: {
                authorization: `Bearer ${this.accessToken}`,
                "x-gw-ims-org-id": this.org,
                "x-ims-org-id": this.org,
                "x-api-key": this.apiKey
            }
        });
```